### DISCUSSION TITLE
[Resolved] Vercel not updating on github push

### DISCUSSION THREAD

**Discussion by kmruban** [vercel/community#5004] [Bug Report]
### Question

I have a vercel project in my hobby plan with vercel that updates everytime my github repo is updated. I have successfully done this so many times and never changed the config of vercel or github. When i push my new code to github it should automatically send to vercel like it has been for months. Today when i push it wont update vercel but it is updating github. Could I have a certain amount of deployments limited with my vercel hobby plan?

**Comment by dmocas** [comment]
What just about to start a new discussion with this.
2 branches commited, 0 builds

**Reply by kmruban** [reply]
What dmocas?

**Reply by kmruban** [reply]
Is this happening for you too?

**Reply by dmocas** [reply]
What you've said. I have been using for months, first time this has happened.
My latest deploy was 3h ago, and now I was updating the branches and it stopped deploying

**Reply by jorgegarba** [reply]
This is happening to me also guys.
Today 3 hours ago as always I made a PR, merged to the main branch. Even Vercel started the automatic deploying but this time I'm seeing in the files section of Vercel stuff that the code is older than the current one on Git Hub.
I even disconnected the GitHub repo and connected it again but with no success ...

**Reply by amyegan** [reply]
@jorgegarba It sounds like the problem you're facing may be different from the [resolved GitHub Webhook issue](https://www.vercel-status.com/incidents/3cgg0wbzpdfw) reported here. Please [start a new discussion](https://github.com/vercel/community/discussions/new) with details about your project and the unexpected behavior you're seeing. That will make the conversation easier for us both to follow 🙏

**Reply by jorgegarba** [reply]
> @jorgegarba It sounds like the problem you're facing may be different from the [resolved GitHub Webhook issue](https://www.vercel-status.com/incidents/3cgg0wbzpdfw) reported here. Please [start a new discussion](https://github.com/vercel/community/discussions/new) with details about your project and the unexpected behavior you're seeing. That will make the conversation easier for us both to follow 🙏 

Yup, fixed now thank you BTW

**Comment by ghost** [comment]
Same here. Commits don't trigger new builds.

edit: Since 16:43 UTC

**Comment by dbejo** [comment]
I experience the same for an hour now.

**Comment by dmocas** [comment]
https://www.vercel-status.com/

They have updated the status, it's broken for now
Deployed with CLI, no issues

**Comment by amyegan** [comment]
Thanks for reporting this issue. There is currently a [problem with Github Webhooks](https://www.githubstatus.com/incidents/dsb3kn1zfdl7). 

https://www.vercel-status.com/incidents/3cgg0wbzpdfw

**Reply by amyegan** [reply]
> What users can expect: “New deployments triggered from GitHub commits are delayed until the incident is resolved” 
> 
> As a workaround, you can still use the Vercel CLI to create deployments: https://vercel.com/docs/cli/deploy

**Reply by dmocas** [reply]
Seems fixed on my side. Just commited and it deployed automatically
